.github/workflows/python-package.yml

```diff
@@ -17,6 +17,10 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Create and activate virtual environment
+      run: |
+        python -m venv .venv
+        source ./.venv/bin/activate
```

Comment:
Just FYI. In GitHub Ations every `run` step is executed in a separate shell. You need to activate your virtual environment for every `run` step. I.e. run `. .venv/bin/activate` first.

Reply:
Without activating the virtual environment in step `Install dependencies` your dependencies will be installed globally but not inside `.venv` dir as you expected. 
Nevertheless `maturin develop` will automatically detect the virtual environment and install `sqlglotrs` package inside it, not globally as other packages. Is that what you expected? Probably not. 

BTW, `maturin develop` is needlessly restrictive by requiring precreated venv. [Quite interesting discussion. ](https://github.com/PyO3/maturin/issues/284)

Reply:
> BTW, maturin develop is needlessly restrictive by requiring precreated venv.

💯 , quite annoying.

> Nevertheless maturin develop will automatically detect the virtual environment and install sqlglotrs package inside it, not globally as other packages.

Ah, this explains my observations. The implicit nature of this behavior is quite annoying as well.

Thanks for catching this! Will fix this accordingly.